aws/table_aws_ssm_inventory.go

```diff
@@ -44,6 +77,198 @@ func tableAwsSSMInventory(_ context.Context) *plugin.Table {
 				Description: "The type of inventory item returned by the request.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "filter_key",
+				Description: "The type of inventory item returned by the request.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getFilterKeyWithOperator, "filter_key"),
+			},
+			{
+				Name:        "filter_value",
+				Description: "The type of inventory item returned by the request.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getFilterKeyWithOperator, "filter_value"),
+			},
+			{
+				Name:        "component_attribute_key",
+				Description: "The attribute key that are supported for type name AWS:AWSComponent, Possible values are: Name,ApplicationType,Publisher,Version,InstalledTime,Architecture and URL.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getFilterKeyWithOperator, "component_attribute_key"),
+			},
+			{
+				Name:        "component_attribute_value",
+				Description: "The value for the component attribute key.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getFilterKeyWithOperator, "component_attribute_value"),
+			},
+			{
+				Name:        "application_attribute_key",
+				Description: "",
```

Comment:
@ParthaI, Could you please add a description for the columns where it is missing?

Reply:
Added the missing description for the columns.